SETUP/UPGRADE.md

```diff
@@ -0,0 +1,151 @@
+# Upgrading
+
+This document covers upgrading an existing installation of DP that
+contains data you want to bring forward to work with the new code
+release.
+
+## Back up your data
+Before doing an upgrade, back up your data. The most important data
+is the one in your database. You can dump that data from within the
+existing installation with:
+```bash
+c/SETUP/dump_db_schema > somewhere/pre_upgrade_db_schema
+```
+
+You should also back up your `projects/` directory.
+
+## Move the old code aside
+Move the old installation directory out of the way. Hang on to it
+for a bit as you may need some files from it.
+
+```bash
+mv c c.old
+```
+
+## Unpack the new DP code
+Unpack the new DP code using either `tar -xf` or `unzip`. This creates
+a new 'c' hierarchy.
+
+## Configure the new DP code
+Follow the 'Installing from scratch' instructions in [README.md](README.md).
```

Comment:
README.md or INSTALL.md?

Reply:
Good catch - thanks! I originally had the file as `README.md` and realized that it really should be `INSTALL.md` and missed this.